client_test.go

```diff
@@ -1115,6 +1119,54 @@ func TestGocloak_GetGroups(t *testing.T) {
 	FailIfErr(t, err, "GetGroups failed")
 }
 
+func TestGocloak_GetGroupsFull(t *testing.T) {
+	t.Parallel()
+	cfg := GetConfig(t)
+	client := NewClientWithDebug(t)
+	token := GetAdminToken(t, client)
+
+	tearDown, groupID := CreateGroup(t, client)
+	defer tearDown()
+
+	groups, err := client.GetGroups(
+		token.AccessToken,
+		cfg.GoCloak.Realm,
+		GetGroupsParams{
+			Full: true,
+		})
+	assert.NoError(t, err, "GetGroups failed")
+
+	for _, group := range groups {
+		if group.ID == groupID {
+			ok := client.UserAttributeContains(group.Attributes, "foo", "alice")
+			assert.True(t, ok, "UserAttributeContains")
+			return
+		}
+	}
+
+	assert.Fail(t, "CreateGroup failed")
```

Comment:
Maybe getgroups failed?

Reply:
Yea, it could be either CreateGroup or GetGroups that failed here. I'm open to whatever you think would be best for the error message. Could also just use the name of the testing function GetGroupsFull.

Reply:
Let’s have GetGroupsFull